inc/sub-headerException.php

```diff
@@ -0,0 +1,124 @@
+<div class="newsSubHeader clearfix">
```

Comment:
The entire theme is heavy on camel case and is part of existing code that I would love to refactor at some point...